tests/dummy/app/templates/snippets/buttons-dropdown.hbs

```diff
@@ -0,0 +1,9 @@
+{{md-btn-dropdown text='Button' activates="dropdown-target"}}
+
+<!-- Dropdown Structure -->
+<ul id='dropdown-target' class='dropdown-content'>
```

Comment:
Imagine that you have several of these button dropdowns on the page at once. You will have several `ul#dropdown-target` elements, and there's really no telling which buttons will target which (or all?).

You need to generate an ID off of something unique to the instance of the component. I like to use the component's root element ID, since that's guaranteed to be unique. See https://github.com/truenorth/ember-cli-materialize/blob/master/addon/components/md-navbar.js#L24-L28